### PR TITLE
Pseudo PR - community review for initial release

### PR DESCRIPTION
**Pull request by ewels**
> _Do not merge - pseudo PR to give review interface for entire pipeline_

Community review for initial release. Aim is to get at least two PR approvals from people in the nf-core community who are not the main developers of this pipeline ✅ 

https://nf-co.re/developers/adding_pipelines#making-the-first-release

### CODE REVIEW

**Review by midopooler — APPROVED** [review]

**Review by JayantGoel001 — APPROVED** [review]

**Review by Surajv311 — APPROVED** [review]

**Review by shivaylamba — APPROVED** [review]

**Review by ArnavBalyan — APPROVED** [review]

**Review by akshitagupta15june — APPROVED** [review]

**Comment by drpatelh** [comment]
Hi Guys! Thank you for finding us and approving this PR 😎! However, we do need some proper, critical reviewing of this PR before the first pipeline release to ensure that pipelines adhere to the highish standards that we have set. This normally comes from a mixture of the core nf-core team as well as other members of the community.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,24 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for the nf-core website
```
Comment (MaxUlysse) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]: I don't think the text here is right. It should be an idea for the nf-core/dualrnaseq pipeline

File: .github/markdownlint.yml [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
-default: true,
+default: true
 line-length: false
 no-duplicate-header:
     siblings_only: true
+no-inline-html:
+    allowed_elements:
+        - img
+        - p
+        - kbd
```
Comment (MaxUlysse) [inline .github/markdownlint.yml:10]: What is this element?

File: Singularity.main [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Bootstrap: docker
```
Comment (MaxUlysse) [inline Singularity.main:1]: I don't this this file is necessary anymore

**Comment by MaxUlysse** [comment]
@javanOkendo any reason for merging this PR? we're still in the middle of processing it.

File: conf/test.config [hunk 4]
```diff
@@ -10,17 +10,54 @@
...
+
+  // run processes
+
+  //run_salmon_selective_alignment = true
+  //run_star = true
+  //run_salmon_alignment_based_mode = true
+  //run_htseq_uniquely_mapped = true
+  //mapping_statistics = true
+
+
```
Comment (MaxUlysse) [inline conf/test.config:30]: leftover comments?

File: conf/test.config [hunk 5]
```diff
@@ -10,17 +10,54 @@
...
+genome_host = 'test_host'
+genome_pathogen = 'test_pathogen'
 }
+
+
+
+
+
+
+
```
Comment (MaxUlysse) [inline conf/test.config:63]:
```suggestion

```

**Pull request merged**